resources/js/components/ui/form.tsx

```diff
@@ -93,7 +93,7 @@ const FormLabel = React.forwardRef<
   return (
     <Label
       ref={ref}
-      className={cn(error && "text-red-500 dark:text-red-900", className)}
```

Comment:
I feel like this might be unnecessary

Reply:
I changed this because the text  on the register form (Username, Email, and Password) blended in with the background color, this makes the text white in dark mode when there are no errors